kuma/javascript/src/payments/components/cancel-subscription-form.jsx

```diff
@@ -0,0 +1,75 @@
+// @flow
+import * as React from 'react';
+import { gettext, interpolate } from '../../l10n.js';
+import ErrorMessage from '../components/error-message.jsx';
+import { deleteSubscriptions } from '../api.js';
+
+type Props = {
+    setShowForm: (((boolean) => boolean) | boolean) => void,
+    onSuccess: () => void,
+    date: string,
+};
+
+export const title = gettext('Are you sure you want to cancel?');
+
+const CancelSubscriptionForm = ({
+    setShowForm,
+    onSuccess,
+    date,
+}: Props): React.Node => {
+    const [status, setStatus] = React.useState<'error' | 'submitting' | 'idle'>(
+        'idle'
+    );
+
+    const handleCancel = () => setShowForm(false);
+
+    const handleSubmit = (event: SyntheticEvent<HTMLFormElement>) => {
+        event.preventDefault();
+        setStatus('submitting');
+
+        // success is handled by parent component
+        const handleSuccess = () => onSuccess();
+        const handleError = () => setStatus('error');
+        deleteSubscriptions(handleSuccess, handleError);
+    };
+
+    if (status === 'error') {
+        return (
+            <p className="alert error" data-testid="error-msg">
+                <ErrorMessage />
+            </p>
+        );
+    }
+
+    return (
+        <form onSubmit={handleSubmit}>
+            <div>
+                <strong>{title}</strong>
+                <p>
+                    {interpolate(
+                        'Your monthly subscription will end on %(date)s. You will have to set up a new subscription if you wish to resume making payments to MDN Web Docs.',
+                        { date }
+                    )}
+                </p>
+                <div className="form-footer">
+                    <button
+                        type="button"
+                        className="cta keep-membership"
```

Comment:
Could we instead use the `.positive` class here?

Reply:
Unfortunately not-- the button in the mockup is outlined in dark gray.

Reply:
On second thought I'll leave as is-- `light-border-color` seems too light and I didn't see any grays that were a better fit.

Reply:
Buttons are next on my list for the design system :)